### ISSUE TITLE
Copy Changes - Home Page + Banner

### ISSUE THREAD

**Issue by rjmajma** [18F/foia-hub#647]
Per a discussion with @vz3 and @ericadeahl, we are going to consider some copy/content changes on these pages. 

Here are the things we considered: 
Home Page
- [ ] Sentence above the search bar
- [ ] Sentence near the tagline that expands on this site being an iterative process and how we are seeking feedback.

Banner
- [ ] Finding a useful way to point back to FOIA.gov

**Comment by rjmajma** [comment]
One suggestion that came from @corizarek was the way we treated the "tagline." On the home page, headers are treated in bold, large font, and underlined. However, the tagline is also treated the same. She suggested we either create a header for it and adjust the text under it or distinguish it some other way from the headers.

**Comment by corizarek** [comment]
Thanks, @rjmajma! Team, I suggest the header openFOIA to be consistent with the rest of that landing page and then take the text "openFOIA jump-starts etc.." and adjust that text to look consistent with other sub-hed text.

**Comment by ericadeahl** [comment]
Here are the updates to the homepage and banner design and copy:

- We added a link to FOIA.gov in the header
- We made the banner much more prominent and added language to make it clear to users that the site is in development and we're looking for feedback. By default the banner will appear open; when it is closed a link appears in the header to reopen it (see second comp below.)
- We moved the headline above the search bar and adjusted the language to actively instruct users what to do. @corizarek I'm using the underline element on this text to give it strong visual emphasis, and the underline is the most distinct part of the openFOIA "brand". Agreed that there isn't much distinction between it and the section titles in the current layout, but I think that placing this text front and center, and on the dark background, helps differentiate it.
- In the "How does the process work?" section, we updated the copy to explain specifically how openFOIA can help users in the FOIA request process, and link to FOIA.gov for more information on the overall FOIA request process.

**openFOIA Homepage with banner open**

![homepage 2](https://cloud.githubusercontent.com/assets/5911280/6738160/5c8ea8ee-ce2d-11e4-8ef6-ce734b1e4803.png)

**openFOIA Homepage with banner closed**

![homepage 1](https://cloud.githubusercontent.com/assets/5911280/6738608/6a1057f8-ce30-11e4-9d48-ac6a39289c3c.png)


**Alternate graphic treatment of the "How does the process work?" section** 
In this version, users click through the steps one by one. We should go ahead and update the text only for now, but it would be great if we could plan on implementing this version in a future sprint.

![homepage 3](https://cloud.githubusercontent.com/assets/5911280/6738558/28292dec-ce30-11e4-85b8-c41707d86d44.png)

@rjmajma
